packages/@okta/vuepress-theme-prose/components/OnThisPage.vue

```diff
@@ -98,7 +98,7 @@ export default {
       this.activeAnchor = onThisPageActiveAnchor
         ? onThisPageActiveAnchor.hash
         : "";
-    }, 200),
+    }, 50),
```

Comment:
Any reason for changing the debounce timeout?

Reply:
I was trying to make this widget work smoother. Actually I think it helped. But I can return it to 200, np